v3/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "codap3",
-  "version": "0.0.1",
+  "version": "v3.0.0-build.700",
```

Comment:
Version listed here should not have the "V" prefix. It's fine to display it that way to the user if desired, but this field is processed by npm and others. I gather the "700" was an attempt to push it beyond the current v2 build number? If that's a goal I'd probably start at "1000" just to make sure they don't overlap, but I'm not sure that actually matters. I was imagining we'd just start low and build.